AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Sync/InitialSync/InitialSyncOperation.swift

```diff
@@ -16,6 +16,7 @@ final class InitialSyncOperation: AsynchronousOperation {
     private weak var reconciliationQueue: IncomingEventReconciliationQueue?
     private weak var storageAdapter: StorageEngineAdapter?
     private let dataStoreConfiguration: DataStoreConfiguration
+    private var modelSyncedPayload: ModelSyncedPayload?
```

Comment:
1. `Payload` isn't terribly meaningful here. I think you're using it to report the result of the initial sync operation, so `OperationResult` might be clearer
2. I think you're using this type to report the status of the entire initial sync operation. Are you using it for other contexts? If not, let's call it something like `InitialSyncOperationResult`. If you are using it elsewhere then pick a name that reflects its intent. "Model synced" is ambiguous: are you talking about emitting one of these each time a model instance is synced, or when the initial sync is finished for a single model type, or when the initial sync is finished for all model types? 
3. Why is this optional? You instantiate it in the initializer, and I don't see you setting it to nil anywhere.

Reply:
2. when `initialSyncOperaion` for each model is finished, emit the event `ModelSynced`. But in my point of view, if it is called `InitialSyncOperationResult` kind of meaning the result should contains a lot of info since `initialSyncOperaion` does more than simply generating a payload? But maybe call `ModelSyncedResult` which is the result of this specific `ModelSynced` Hub event? What do you think?
3. Ya, this can be non-optional. Let me change it